frontend/components/sorting-buttons/component.tsx

```diff
@@ -65,11 +67,24 @@ export const SortingButtons: FC<SortingButtonsProps> = ({
           onAction={(key: string) => onChange({ sortBy: key })}
           onOpen={() => setSortByMenuOpen(true)}
           onClose={() => setSortByMenuOpen(false)}
+          className="min-w-fit"
         >
           <MenuSection>
-            {options.map(({ key, label }) => (
-              <MenuItem key={key}>{label}</MenuItem>
-            ))}
+            {options.map(({ key, label }) => {
+              const isSelected = selectedSortByOption.key === key;
+              return (
+                <MenuItem key={key}>
+                  <span
+                    className={cx({
+                      'text-green-dark': isSelected,
+                    })}
+                  >
+                    {label}
+                    {isSelected && <Icon width={18} className="inline ml-2" icon={Check} />}
+                  </span>
+                </MenuItem>
+              );
+            })}
```

Comment:
You don't need to make any of these changes here. The `<Menu />` component receives a prop named `expandedKeys` which, if defined, must contain the list of the active menu items. The active items will automatically get a green text colour and check icon.

```tsx
<Menu
  expandedKeys={[selectedSortByOption.key]}
  // The rest of your props here
/>
```